platform/darwin/src/MGLAccountManager.h

```diff
@@ -1,11 +1,12 @@
-#import <Foundation/Foundation.h>
```

Comment:
As mentioned in that jazzy ticket, this header itself must import Foundation.h. It's unfortunate, but that shouldn't pose a problem for the visibility changes you're making, should it?